airflow/www/templates/airflow/dag.html

```diff
@@ -84,8 +84,17 @@ <h4 class="pull-right" style="user-select: none;-moz-user-select: auto;">
         {{ state_token }}
       {% endif %}
       <a class="label label-default" href="{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id={{ dag.dag_id }}">
-        schedule: {{ dag.schedule_interval }}
+        Schedule: {{ dag.schedule_interval }}
       </a>
+      {% if dag_model is defined and dag_model.next_dagrun_create_after is defined %}
+        <p class="label label-default" style="margin-left: 5px">
+          Next Run: <span id="next-run">{{ dag_model.next_dagrun_create_after }}</span>
```

Comment:
By making this a `<time>` element it will automatically get converted by the [timezone selector](https://github.com/apache/airflow/blob/e4d3da16bad66aa40549cedac325d02e6b6559a9/airflow/www/static/js/datetime_utils.js#L74) (not to mention that just semantically this _is_ a time :grin:)

Reply:
Oh yes, I forgot that we select elements by `<time>` in `datetime_utils.js`. Fixed.